bitnami/phpbb/templates/NOTES.txt

```diff
@@ -1,54 +1,84 @@
-{{- if or .Values.mariadb.enabled .Values.externalDatabase.host -}}
-
-1. Get the phpBB URL by running:
 
-{{- if contains "NodePort" .Values.service.type }}
+{{- if or .Values.mariadb.enabled .Values.externalDatabase.host -}}
+** Please be patient while the chart is being deployed **
+{{- if and .Values.ingress.enabled (ne .Values.service.type "ClusterIP") }}
+** Notice : Usually with ingress the service.type should be set to ClusterIP, which is not the case to this deployment! **
+{{- end }}
 
-  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services "{{ template "phpbb.fullname" . }}")
+1. Access you phpBB instance with:
+
+{{- if .Values.ingress.enabled }}
+  {{ if .Values.ingress.tls }}
+    {{- range .Values.ingress.hosts }}
+    - https://{{ . }}
+    {{- end }}
+  {{- else }}
+    {{- range .Values.ingress.hosts }}
+    - http://{{ . }}
+    {{- end }}
+  {{- end }}
+{{- else if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "phpbb.fullname" . }})
   export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
-  echo "phpBB URL: http://$NODE_IP:$NODE_PORT/"
+  echo "phpBB! URL: http://$NODE_IP:$NODE_PORT/"
```

Comment:
I see many occurrences of "phpBB!" as if the product name includes the "!", it does not look to be the case so could you remove it?